### PR TITLE
commit the first work in ehm

### PR DESCRIPTION
**Pull request by victoria1999531**

### CODE REVIEW

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class FundamentalTest {
+    /**
+     * The number and  abbreviation  of the month
+     */
+    public static final int Month = 10;
+ //1.使用Intellij 创建一个包，包名为ehm,创建一个叫做FundamentalTest的java文件，在其中打印“hello world”。
```
Comment (loverogersun) [inline src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: PR1: Please try to use English in code

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                break;
+        }
+    }
+  /*
+  4.定义一个方法，这个方法在main函数中调用，在main函数中定义一个字符串“Good Morning”，
+  把字符串作为参数传入定义的方法中，如果字符串含有空格，并且字符串第一个字母为G，
+  则把这个字符串用空格进行分割，将分割后的两个单词打出来,并且一个单词为一行（使用for each的形式），
+  然后将“Good Morning”的长度作为这个方法的返回值，最后在main函数中判断如果长度为偶数，
+  则打印这个字符串的长度，否侧输出“字符串为奇数”
+   */
```
Comment (loverogersun) [inline src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: PR1: 1) Good JavaDoc though, the format for JavaDoc is not correct, please correct them.  2) Please try to use English

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+ * Copyright 2020 Kingland Systems Corporation. All Rights Reserved.
+
+ */
+package com.kingland.eip.ehm;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class FundamentalTest {
```
Comment (loverogersun) [inline src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: PR1: Let's move test code to test folder, and let's use junit to write unit test code

File: src/main/java/com/kingland/eip/ehm/LogTest/BaseLog.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+/*
+
+ * Copyright 2020 Kingland Systems Corporation. All Rights Reserved.
+
+ */
+package com.kingland.eip.ehm.LogTest;
+
+public class BaseLog {
```
Comment (loverogersun) [inline src/main/java/com/kingland/eip/ehm/LogTest/BaseLog.java:8]: PR1: Add JavaDoc for class

File: src/main/java/com/kingland/eip/ehm/LogTest/Main.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package com.kingland.eip.ehm.LogTest;
+
+public class Main {
```
Comment (loverogersun) [inline src/main/java/com/kingland/eip/ehm/LogTest/Main.java:3]: This is not a good class name, consider to change it

**Review by loverogersun — COMMENTED** [review]
Let's create gradle Java project, using gradle to manage Java lib dependencies.

**Review by loverogersun — COMMENTED** [review]
PR1: Very Important: Please add meaningful message in your commits.

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java (hunk 1 above)
Comment (victoria1999531) [reply src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: Thanks.i'll solve it.

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java (hunk 2 above)
Comment (victoria1999531) [reply src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: Thanks.i'll solve it.

File: src/main/java/com/kingland/eip/ehm/FundamentalTest.java (hunk 3 above)
Comment (victoria1999531) [reply src/main/java/com/kingland/eip/ehm/FundamentalTest.java]: Thanks.i'll solve it.

File: src/main/java/com/kingland/eip/ehm/LogTest/BaseLog.java (hunk 4 above)
Comment (victoria1999531) [reply src/main/java/com/kingland/eip/ehm/LogTest/BaseLog.java:8]: Thanks.i'll solve it.

File: src/main/java/com/kingland/eip/ehm/LogTest/Main.java (hunk 5 above)
Comment (victoria1999531) [reply src/main/java/com/kingland/eip/ehm/LogTest/Main.java:3]: Thanks.i'll solve it.

**Pull request merged**